ibm/resource_ibm_iam_service_policy.go

```diff
@@ -150,10 +150,29 @@ func resourceIBMIAMServicePolicy() *schema.Resource {
 			},
 
 			"tags": {
-				Type:     schema.TypeSet,
```

Comment:
General question. Do these changes deprecate the previous support for tags?

I think there is a separate type of tag for policies (I think resource tag or user tag, the CLI refers to them as "Access tags for the resource"). I'm pretty sure this is different from the way we (IAM) implements Access Management tags for a policy.

See:
```sh
$ ibmcloud iam user-policy-create --help
...
USAGE:
  ibmcloud iam user-policy-create USER_NAME {--file JSON_FILE | --roles ROLE_NAME1,ROLE_NAME2... [--service-name SERVICE_NAME] [--service-instance SERVICE_INSTANCE_GUID] [--region REGION] [--resource-type RESOURCE_TYPE] [--resource RESOURCE] [--resource-group-name RESOURCE_GROUP_NAME] [--resource-group-id RESOURCE_GROUP_ID] [--tags name1:value1,name2:value2...] [--account-management] [--attributes name=value,name=value...]} [--output FORMAT] [-q, --quiet]
...
OPTIONS:
...
  --tags value                 Access tags of the resource.
...
```

I remember there being an issue as to how to resolve these two types of tags, but I don't remember if there is an elegant solution. 
@hkantare Do you know about the way tags is currently implemented?

Reply:
@swcolley , 
the current tags were supported based on one of the user request.. since IAM doesnt support any tags like other services..user wanted to differentiate resources based on tags..

this tags will just be saved in the statefile and does nothing else..
Changing this argument will break existing users who are using it.. I would suggest to use new argument something like `resource_tags`